plotly/Cargo.toml

```
@@ -40,7 +40,7 @@ wasm-bindgen-futures = { version = "0.4", optional = true }
 
 [dev-dependencies]
 csv = "1.1.6"
-image = "0.24.4"
+image = "0.25.1"
```

```suggestion
image = ">=0.24, <0.26"
```
Depend on either versions 0.24 or 0.25, as either works.

---

0.25 doesn't work. See previous comment.

---

@sjackman , it was working, you were correct. I just missed to update the TOML file in the examples. Cargo was trying to be smart with finding a common version that worked.